.github/workflows/curator-service-package.yml

```diff
@@ -17,15 +17,34 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: docker/build-push-action@v1
+      -
+        name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=${{github.repository}}/curatorservice
+          VERSION=edge
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/}
+          elif [[ $GITHUB_REF == refs/heads/* ]]; then
+            VERSION=$(echo ${GITHUB_REF#refs/heads/} | sed -r 's#/+#-#g')
+          elif [[ $GITHUB_REF == refs/pull/* ]]; then
```

Comment:
This won't be called as we are not building on PR, but okay to leave it I guess

Reply:
Yup, I pasted that from the example docs and just deleted the outputs that I don't think we need, missed that bit.